apps/routes/test/populate_caches_test.exs

```diff
@@ -26,7 +26,7 @@ defmodule Routes.PopulateCachesTest do
   alias __MODULE__.FakeRepo
 
   setup do
-    {:ok, _} = Agent.start_link(&MapSet.new/0, name: FakeRepo)
+    _ = Agent.start_link(&MapSet.new/0, name: FakeRepo)
```

Comment:
**question:** is there a particular reason you were no longer able to match on `{:ok, _}`? At first glance it doesn't look like this is related to other changes as far as I can tell.

Reply:
Oh, yeah I no longer remember the why behind it, it likely works fine as is.

Reply:
I personally have a general preference towards being as strict as possible with matches, but I suppose it's test code that's part of the setup as opposed to an actual test so it doesn't actually change either application functionality or what the test suite is testing.